src/components/CommentCard.jsx

```diff
@@ -20,13 +22,21 @@ const CommentCard = ({ review }) => {
         setError(err);
         setIsLoading(false);
       });
-  }, []);
+  }, [commentSubmitted]);
```

Comment:
we don't want to re-fetch all the comments when we submit a new one really, as this creates an odd looking experience for the user - also, what if everyone else deleted their comments? What if in that time 100 new comments were added? It would feel a bit strange.

This is where we want to use optimistic rendering to add it to our comments _in state_ and post the comment to the API.